Catroid.iml

```diff
@@ -1,4 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<module external.linked.project.path="$MODULE_DIR$" external.root.project.path="$MODULE_DIR$" external.system.id="GRADLE" external.system.module.group="" external.system.module.version="unspecified" type="JAVA_MODULE" version="4">
-</module>
-
+<module external.linked.project.id="Catroid" external.linked.project.path="$MODULE_DIR$" external.root.project.path="$MODULE_DIR$" external.system.id="GRADLE" external.system.module.group="" external.system.module.version="unspecified" type="JAVA_MODULE" version="4">
```

Comment:
As @thmq  said I have run the gradle task:git./gradlew gitSkipWorktreeForIntellijConfigFiles before doing any commits which should ignore the files need to be ignored. Please guide me how do I exclude these files?